tv_apps/smart-home/style/home.css

```diff
@@ -71,8 +71,8 @@ section {
   top: -23.14rem;
   width: 100%;
   height: 14.926rem;
-  display: none;
   text-align: center;
+  z-index: -1;
 }
 
 .card-panel smart-button {
```

Comment:
If card-panel is only for edit mode, we should move these css to `edit.css` instead of `home.css`